src/content/en/updates/2019/02/intersectionobserver-v2.md

```diff
@@ -0,0 +1,210 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Augment IntersectionObserver to report information about occlusion and visual effects.
+
+{# wf_updated_on: 2019-02-11 #}
+{# wf_published_on: 2019-02-06 #}
+{# wf_tags: intersectionobserver #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: Lorem ipsum. #}
+{# wf_blink_components: Blink>Layout #}
+
+# Trust is Good, Observation is Better—Intersection Observer&nbsp;v2 {: .page-title}
+
+{% include "web/_shared/contributors/thomassteiner.html" %}
+
+<div class="clearfix"></div>
+
+Intersection Observer&nbsp;v1 is one of these APIs that's probably universally loved, and, now that
+[Safari supports it](https://webkit.org/blog/8582/intersectionobserver-in-webkit/) as well,
+it's also finally universally usable in all major browsers. For a quick refresher of the API,
+I recommend watching [Surma](../../../resources/contributors/surma)'s
+[Supercharged Microtip](https://www.youtube.com/embed/kW_atFXMG98) on Intersection
+Observer&nbsp;v1—also embedded below for your viewing pleasure—or reading Surma's in-depth
+[article](../../2016/04/intersectionobserver).
+People have used Intersection Observer&nbsp;v1 for a wide range of use cases, like, for example,
```

Comment:
```suggestion
People have used Intersection Observer&nbsp;v1 for a wide range of use cases like
```

"like, for example," is very conversational but a bit awkward in writing. Recommend simply removing "for example,"

Reply:
Edited, thanks!